From 3a5815c905e9a38af2f2998db0da74ef5f3b0949 Mon Sep 17 00:00:00 2001
From: Guneshwor Singh <guneshwor.o.singh@intel.com>
Date: Thu, 7 Sep 2017 09:58:03 +0530
Subject: [PATCH 0778/2313] ASoC: Intel: board: Remove unused function
 cnl_dmic_fixup

cnl_dmic_fixup is never used, remove it to shun compiler warning.

Signed-off-by: Guneshwor Singh <guneshwor.o.singh@intel.com>
---
 sound/soc/intel/boards/cnl_cs42l42.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/sound/soc/intel/boards/cnl_cs42l42.c b/sound/soc/intel/boards/cnl_cs42l42.c
index 3a1bcbf..c444535 100644
--- a/sound/soc/intel/boards/cnl_cs42l42.c
+++ b/sound/soc/intel/boards/cnl_cs42l42.c
@@ -187,16 +187,6 @@ static int cnl_cs42l42_codec_fixup(struct snd_soc_pcm_runtime *rtd,
 	return ret;
 }
 
-static int cnl_dmic_fixup(struct snd_soc_pcm_runtime *rtd,
-				struct snd_pcm_hw_params *params)
-{
-	struct snd_interval *channels = hw_param_interval(params,
-						SNDRV_PCM_HW_PARAM_CHANNELS);
-	channels->min = channels->max = 2;
-
-	return 0;
-}
-
 static const struct snd_soc_pcm_stream cnl_cs42l42_dai_params = {
 	.formats = SNDRV_PCM_FMTBIT_S24_LE,
 	.rate_min = 48000,
-- 
2.7.4

